docs/understanding-radicle/faq.md

```diff
@@ -1,14 +1,139 @@
+whenever your data is published to the network, it can be replicated and stored
+on another peer's machine.
+
+## Can I create private repositories on Radicle?
+No, not yet - but in the future! Private projects with end-to-end encryption are
+on our roadmap. In the meantime, be sure to note that everything you put on
+Radicle will be publicly available.
+
+## What is a remote?
+A 'remote' refers to a version of your project that is maintained by another
+person. To collaborate with others on Radicle, you have to add and follow other
+people's remotes to be able to fetch changes from them. You can manage remotes
+on your project page (See [Adding Remotes][ar]). For more on how remote
+repositories work, see the [Git documentation][mr]]. 
+
+## What's a Radicle ID?
+A Radicle ID is a unique way to identify projects in the Radicle
+Network. You use a project's Radicle ID to find it on Radicle Upstream.
+
+## What's a Device ID?
+A Device ID is the encoding of a peer's public key tied to a specific device.
+People will be able to manage multiple Device IDs in the future, but for now can
+only have one Device ID per identity.
+
+## What does `following` mean in Radicle?
+Following a project replicates its data to your
+machine. This allows the follower to subscribe to updates from the project's
+maintainer(s) or other remotes. It is also a way to signal interest in the
+project or peer by further replicating the data across their network, making it
+available to other people on the network. See [Tracking][tr].
+
+## Can I use Radicle with multiple devices?
+Yes and no. While there isn't multi-device support yet, you can still create
+accounts on different devices, they just won't be linked under one Upstream user
+account.
+
+## How do I make sure nobody else has my display name?
+You can't.... yet. We will be introducing unique names soon 👍
+
+(P.S. While your display name isn't unique, your emoji avatar is!)
+
+## What happens if I forget my passphrase?
+Without your passphrase, there is no way to grant the Upstream client access to
+your secret key. This means that without your passphrase, there is no way to
+access or publish data to the Radicle network - so make sure you keep it safe!
+
+## I ran into a issue, where can I report it?
+Please submit a support request our #support channel, or submit an issue on our
+GitHub repository.
+
+## I need some help, where do I reach out?
+For help, join our #support channel in our Matrix chat or in the Help category
+of our Discourse.
+
+
+[ar]: /using-radicle/pushing-and-pulling-changes#adding-remotes
+[cp]: /using-radicle/creating-and-sharing-projects.md
+[gs]: /getting-started/getting-started.md
+[hw]: /understanding-radicle
+[tr]: /how-it-works.md/#tracking
+
+[ls]: https://radicle.community/t/radicle-licensing-model/282/8
+[mr]: https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes
+[sm]: LINK TO RADICLE>COMMUNITY POST
```

Comment:
This breaks all the links!

Reply:
I know I just realized that. I will sub it for a dummy link for now.